pandas/tests/groupby/test_categorical.py

```diff
@@ -433,6 +433,35 @@ def test_observed_groups_with_nan(observed):
     tm.assert_dict_equal(result, expected)
 
 
+def test_observed_ops(agg_func):
+    cat = pd.Categorical(['a', np.nan, np.nan], categories=['a', 'b', 'c'])
+    ser = pd.Series([1., 2., 3.])
+    df = pd.DataFrame({'cat': cat, 'ser': ser})
+
+    grp = df.groupby('cat', observed=False)['ser']
+    func = getattr(grp, agg_func)
+
+    if agg_func == 'nth':  # Need an argument
```

Comment:
I would do nth separately, its not similar to the other reduction functions, and the test becomes much more straightforward.